cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/schedule/ScheduleTaskRunner.java

```diff
@@ -88,6 +88,7 @@ public void launch(Job job) throws Exception {
 
     TriggeringScheduleInfo triggeringScheduleInfo = getTriggeringScheduleInfo(job);
     systemArgs.put(ProgramOptionConstants.TRIGGERING_SCHEDULE_INFO, GSON.toJson(triggeringScheduleInfo));
+    lifecycleService.addAppCDAPVersion(programId, systemArgs);
```

Comment:
Is this needed? It seems like it would be added later in the lifecycleService.runInternal() call.

Reply:
Removed